### DISCUSSION TITLE
在win10上找不到doxygen

### DISCUSSION THREAD

**Discussion by happyxianyu** [xmake-io/xmake#2357] [Q&A]
已经安装了doxygen，也设置了path环境变量，但运行"xmake doxygen"后，显示"error: doxygen not found!"

**Comment by waruqi** [comment]
切到非工程目录，执行 xmake l find_tool doxygen 看下

**Reply by happyxianyu** [reply]
显示"error: cannot import module: find_tool, not found!"

**Reply by waruqi** [reply]
xmake l lib.detect.find_tool doxygen

**Reply by happyxianyu** [reply]
{
  name = "doxygen",
  program = "doxygen"
}

解决了，谢谢
